src/FacultyDirectory/ClientApp/src/components/InputArray.tsx

```diff
@@ -1,4 +1,26 @@
 import React, { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+interface IActiveIndicatorProps {
+    hasValue: boolean
+}
+
+export function ActiveIndicator(props: IActiveIndicatorProps) {
+    let classNames = "active-color-indicator"
+    if(props.hasValue) {
+        classNames += " active"
+    }
+    return (
+        <div className={classNames} />
+    )
+}
+
+export function useIndicator(initialValue: string) {
```

Comment:
Is this being used anywhere?

Reply:
not right now, I think I wanted to generate a handler to be bound to onChange in case the data wasn't stored somewhere else